src/renderer/components/+catalog/catalog.tsx

```diff
@@ -83,9 +83,9 @@ export class Catalog extends React.Component {
         <MenuItem key="remove-from-hotbar" onClick={() => this.removeFromHotbar(item) }>
           <Icon material="clear" interactive={true} title="Remove from hotbar"/> Remove from Hotbar
         </MenuItem>
-        { this.contextMenu.menuItems.map((menuItem) => {
+        { this.contextMenu.menuItems.map((menuItem, index) => {
           return (
-            <MenuItem key={menuItem.title} onClick={() => menuItem.onClick()}>
+            <MenuItem key={`menuitem-${index}`} onClick={() => menuItem.onClick()}>
```

Comment:
Why switch this, also this is no different that using the index as the key. Is there a reason to do this formatting on every render?

Reply:
No reason, changed it to use plain `index`.